src/orchestrator.ts

```diff
@@ -21,80 +22,100 @@ type OrchestratorOptions = {
   commandPort: number;
   connectionPort: number;
   agentPort: number;
-  delegate?: Mailbox;
   testFiles: [string];
   testManifestPath: string;
   testFilePort: number;
 }
 
 export function* createOrchestrator(options: OrchestratorOptions): Operation {
-  let atom = new Atom();
   console.log('[orchestrator] starting');
 
-  let mail = options.delegate || new Mailbox();
+  let atom = new Atom();
+
+  let proxyServerDelegate = new Mailbox();
+  let commandServerDelegate = new Mailbox();
+  let connectionServerDelegate = new Mailbox();
+  let agentServerDelegate = new Mailbox();
+  let appServerDelegate = new Mailbox();
+  let manifestGeneratorDelegate = new Mailbox();
+  let manifestServerDelegate = new Mailbox();
```

Comment:
This is the downside of this PR, we have to create a whole bunch of mailboxes. I think this is worth it anyway for the added clarity.